app/src/androidTest/java/com/example/admin/personallibrarycatalogue/data/TestDb.java

```diff
@@ -0,0 +1,96 @@
+package com.example.admin.personallibrarycatalogue.data;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.test.AndroidTestCase;
+import android.util.Log;
+
+import com.example.admin.personallibrarycatalogue.R;
+import com.example.admin.personallibrarycatalogue.data.DatabaseContract.BooksTable;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+
+/**
+ * Created by Admin on 26.05.2015.
+ */
+public class TestDb extends AndroidTestCase {
+
+    private static final String BOOK_TITLE = "Thinking in Java";
+    private static final String BOOK_AUTHOR = "Bruce Eckel";
+    private static final String BOOK_DESCRIPTION = "Thinking in Java is a book about the Java programming language, written by Bruce Eckel and first published in 1998";
+    public static final String LOG_TAG = TestDb.class.getSimpleName();
+
+
+    public void testCreateDb() throws Throwable {
+        mContext.deleteDatabase(LibraryDatabaseHelper.DATABASE_NAME);
+        SQLiteDatabase database = new LibraryDatabaseHelper(
+                this.mContext).getWritableDatabase();
+        assertEquals(true, database.isOpen());
+        database.close();
+    }
+
+    public void testInsert() {
+        LibraryDatabaseHelper helper = new LibraryDatabaseHelper(mContext);
+        SQLiteDatabase database = helper.getWritableDatabase();
+
+        ContentValues book = createBooksValues();
+        long bookId = database.insert(BooksTable.TABLE_NAME, null, book);
+
+        assertTrue(bookId != -1);
```

Comment:
https://www.dropbox.com/s/v80zp9j7kqxpxoc/Screenshot%202015-06-10%2012.55.22.png?dl=0
![screenshot 2015-06-10 12 55 22](https://cloud.githubusercontent.com/assets/1070647/8079838/fddf8e88-0f6f-11e5-8f77-b787a891ef51.png)

Just FYI. Here you use row ```-1``` value for assert. It's a good idea cause